junit-jupiter-api/src/main/kotlin/org/junit/jupiter/api/Assertions.kt

```diff
@@ -86,6 +102,201 @@ fun assertAll(vararg executables: () -> Unit) =
 fun assertAll(heading: String?, vararg executables: () -> Unit) =
     assertAll(heading, executables.toList().stream())
```

Comment:
Reading this on GitHub makes me confused about the return type.
Again, not your code, but all your return types are nice and clean, probably worth aligning.

Reply:
I've noticed that return type is not specified only for one-liners that return `Unit`. I assume, to resemble usual methods, where it isn't mandatory to specify `Unit` return type. For one-liners that return anything besides `Unit`, a return type is already specified